ui/raidboss/data/06-ew/raid/p7n.txt

```diff
@@ -5,3 +5,120 @@ hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+# -ii 77F6 77F7 77FA 77FD 77FF 7800 7804 7806 7807 780A 783E 7841
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+3.0 "--sync--" sync / 1[56]:[^:]*:Agdistis:78E3:/
+15.8 "Bough of Attis (OUT)" sync / 1[56]:[^:]*:Agdistis:77F9:/ window 16,5
```

Comment:
This is a tiny nit, but can you make these lowercase to match with the left/right?

Reply:
I started with them that way, but `(in)/(out)` is kind of minimally readable on the timeline bars. It's possible that's just my UI scale being weird, but that's why I uppercased them. Not a big deal if the consistency outweighs readability when there's a trigger for it.